ngraph/frontend/CMakeLists.txt

```diff
@@ -2,6 +2,52 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if(NOT WIN32)
+    # DEBUG - print all variables
+    # get_cmake_property(_variableNames VARIABLES)
+    # set(ALL_VARS "")
+    # foreach (_variableName ${_variableNames})
+    #     set(ALL_VARS ${ALL_VARS} -D${_variableName}=${${_variableName}}\ )
+    # endforeach()
+    # message(---------------------ALL VARS: ${ALL_VARS}-------)
+
+    message(${CMAKE_CURRENT_SOURCE_DIR}/cmake_static_protobuf)
+    message(BINARY ${CMAKE_CURRENT_BINARY_DIR})
+
+    # There seems no suitable other way to identify exact output binary name for libprotobuf
+    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+        set(PROTOBUF_STATIC_LIB_OUTPUT ${CMAKE_ARCHIVE_OUTPUT_DIRECTORY}/libprotobufd.a)
```

Comment:
and `CMAKE_STATIC_LIBRARY_PREFIX`, `CMAKE_STATIC_LIBRARY_SUFFIX`

Reply:
Done. However was not able to use CMAKE_DEBUG_POSTFIX. This is because on linux it is empty (only on Windows it is 'd'), but protobuf has hardcoded its 'debug' name as libprotobufd for all platforms